packages/neon/neon/lib/src/widgets/image_wrapper.dart

```diff
@@ -3,31 +3,35 @@ import 'package:flutter/material.dart';
 class NeonImageWrapper extends StatelessWidget {
   const NeonImageWrapper({
     required this.child,
-    required this.color,
+    this.color,
     this.size,
     this.borderRadius,
     super.key,
   });
 
   final Widget child;
-  final Color color;
+  final Color? color;
   final Size? size;
   final BorderRadius? borderRadius;
 
   @override
-  Widget build(final BuildContext context) => ClipRRect(
-        borderRadius: borderRadius ?? BorderRadius.zero,
-        child: ColorFiltered(
-          colorFilter: ColorFilter.mode(color, BlendMode.dstATop),
-          child: SizedBox.fromSize(
-            size: size,
-            child: ColoredBox(
-              color: Colors.transparent,
-              child: Center(
-                child: child,
-              ),
+  Widget build(final BuildContext context) {
+    final color = this.color ?? Theme.of(context).colorScheme.secondaryContainer;
```

Comment:
I don't think this is necessary. For most images you want a white background regardless of the theme, otherwise many images will not be readable. You could argue that white is usually used as a background because of the dominance of light themes, but it doesn't matter since we just want to display the images in the best way possible